File: Day2/datatypes.js
function data_type (input){
	if (typeof input === 'string'){
		output = input.length;
	}

	else if (typeof input === 'undefined'){
		output = "No Value";

	}
	
	else if (typeof input === 'boolean'){
		output = boolean(input);

	}

	else if (typeof input === 'number'){
		if (input < 100){
			output = "less than 100";
		}
		else{
			output = "more than 100";
		}
	}

	else if (value instanceof input){
		output = input[3];

	}

	else (input  === 'function'){
		output = input(true);

	}

	return output;

}

File: DataTypes.py
def data_type(data):
    if type(data) == str:
        return len(data)
    elif type(data) == bool:
        return data
    elif type(data) == list:
        if len(data) < 3:
            return None
        else:
            return data[2]
    elif data == None:
        return 'no value'
    elif type(data) == int:
        if data < 100:
            return'less than 100'
        elif data > 100:
            return'greater than 100'
